### PR TITLE
Document missing docstrings

### PR DESCRIPTION
**Pull request by tseclaudia**
## Description

Addresses #1961 
## Checklist

[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [ ] Gallery example in `./doc/examples` (new features only)
- [ ] Unit tests

[For detailed information on these and other aspects see [scikit-image contribution guidelines](http://scikit-image.org/docs/dev/contribute.html)]
## References

[If this is a bug-fix or enhancement, it closes issue # ]
[If this is a new feature, it implements the following paper: ]

### CODE REVIEW

File: skimage/measure/_regionprops.py [hunk 1]
```diff
@@ -130,6 +135,11 @@ def centroid(self):
 
     @only2d
     def convex_area(self):
+    """
+    Returns
+    _______
```
Comment (stefanv) [inline skimage/measure/_regionprops.py]: Not underscores, dashes

File: skimage/measure/_regionprops.py [hunk 2]
```diff
@@ -110,6 +110,11 @@ def __init__(self, slice, label, label_image, intensity_image,
 
     @_cached
     def area(self):
+    """
+    Returns
+    -------
+    The number of pixels of the region image as an integer.
```
Comment (stefanv) [inline skimage/measure/_regionprops.py]: I think these descriptions should go in the `regionprops` docstring.

**Comment by soupault** [comment]
@tseclaudia Hi! Do you have an intent to finalize this PR? It would be a very nice addition to the upcoming release.

**Comment by tseclaudia** [comment]
@soupault I made another pull request in #2351 with additional changes and a clearer title that addresses the same issue. Sorry. I didn't know how to delete this pull request. :( 
I still can't locate the source of the error that's causing the travis-ci check to fail though.

**Comment by soupault** [comment]
Closing by request.

**Pull request closed without merging**